### ISSUE TITLE
Does this also work with Aroma?

### ISSUE THREAD

**Issue by supercrooku** [koolkdev/wiiu-nanddumper#5]
I really really want to use this, is it compatible with Aroma?

**Comment by DimitriPilot3** [comment]
Yes. You can download it as a payload (`Nanddumper`) on the Aroma website. Refer to the current https://wiiu.hacks.guide for instructions; it has an entire section about making a NAND backup using that.

To access the payload loader, hold B on the wiiu gamepad while launching Aroma (i.e., during system boot or while using the browser exploit).
